apps/src/code-studio/pd/facilitator_program_registration/FacilitatorProgramRegistration.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
```

Comment:
Indeed! The intent of both this and the `ProgramRegistrationComponent` is that they should be able to be trivially converted into something like a `FormHandler` and `FormComponent` pair of classes, for generalizable form creation.

I just plan to do that generalization in a separate PR